src/app/TradeApp/index.tsx

```diff
@@ -20,25 +21,37 @@ const Trading = React.lazy(
     ),
 )
 
-const Menu: React.FC = () => (
-  <Frame>
-    <footer>Menu</footer>
-  </Frame>
-)
-
 const NavTools: React.FC = () => (
   <Frame>
     <footer>Tools</footer>
   </Frame>
 )
 
-export const TradeApp: React.FC = () => (
-  <GenericLayout menu={<Menu />} navTools={<NavTools />}>
-    <React.Suspense fallback={null}>
-      <Switch>
-        <Route path="/v2" exact component={Trading} />
-        <Route component={NotFound} />
-      </Switch>
-    </React.Suspense>
-  </GenericLayout>
-)
+export const TradeApp: React.FC = () => {
+  const menu = (
+    <Menu>
```

Comment:
we already have code for this no? NavLinks or sth

Reply:
Well i guess this is a _generic menu_ but here is the code I am [referring to](https://github.com/gnosis/dex-react/blob/v2-add-menu/src/components/layout/LegacyTradeLayout/Header/Navigation.tsx)

Reply:
Good point. I'm refactoring the new design. The code u linked seems a bit overcomplicated, maybe is good to add it, or maybe we don't need it so complex for v2, I'll add a task for the TODO to review this

Reply:
I'll look it over rn